marc_to_solr/lib/traject_config.rb

```diff
@@ -1074,15 +1082,31 @@
   end
 end
 
+to_field 'lc_pipe_facet' do |record, accumulator|
+  delimiter = '|'
```

Comment:
Can we use a multi-character delimiter (e.g. `|||`)?  I just dread the day when the data gets a value with a pipe, and we have to do this again.